### ISSUE TITLE
Not all Android versions do now log

### ISSUE THREAD

**Issue by peterdk** [Nercury/android_logger-rs#66]
I upgraded from 0.12.0 to 0.13.0, and while logging still works at API 33, I get with indentical apk, no output in API 26 for example. 
I don't know why? Could it be not selecting proper output log buffer? I basically do this: 
```
 android_logger::init_once(Config::default().with_max_level(LevelFilter::Trace));
```

**Comment by peterdk** [comment]
Thanks for the library by the way!

**Comment by tyranron** [comment]
@peterdk maybe... did you try to specify log buffer explicitly?

**Comment by tyranron** [comment]
@peterdk 

> I would expect that the LogId is set to System by default?

No, of course not. By [default](https://github.com/Nercury/android_logger-rs/blob/v0.13.0/src/lib.rs#L131), [system defaults](https://docs.rs/android_log-sys/latest/src/android_log_sys/lib.rs.html#47) are used. I guess that it differs depending on Android version.

Prior 0.13, we've used [`__android_log_write()`](https://developer.android.com/ndk/reference/group/logging#group___logging_1ga32a7173b092ec978b50490bd12ee523b), and now we use [`__android_log_buf_write()`](https://developer.android.com/ndk/reference/group/logging#group___logging_1gafe9405c7503ba5cee092ed5a54873231). Its documentation says:
> Apps should use [__android_log_write()](https://developer.android.com/ndk/reference/group/logging#group___logging_1ga32a7173b092ec978b50490bd12ee523b) instead.

So, I can conclude that [`__android_log_write()`](https://developer.android.com/ndk/reference/group/logging#group___logging_1ga32a7173b092ec978b50490bd12ee523b) is hardcoded to use [`LOG_ID_MAIN` buffer](https://developer.android.com/ndk/reference/group/logging#log_id).

I guess you should be totally OK with `.with_log_buffer(LogId::Main)`.

**Comment by peterdk** [comment]
Ah ok I understand. Just noting that lots of users probably now get their logs not visible by default. Maybe update documentation for this?

**Comment by tyranron** [comment]
Ideally, it would be nice to default to `LOG_ID_DEFAULT` on APIs supporting it and `LOG_ID_MAIN` otherwise. However, I cannot figure out how to detect API level programmatically.

@peterdk could you advise something here? Or maybe you can even craft a PR doing that?

**Comment by peterdk** [comment]
What is your reasoning behind not just setting it to default main? I haven't come across any other logcat log in my day to day Android development? And in case some platform Android Rust code uses this library, they can set it to the preferred stream? As you wrote, normal apps only use Main. I didn't even know there were others.

**Comment by tyranron** [comment]
rfc @rsglobal

**Comment by tyranron** [comment]
@peterdk 

> As you wrote, normal apps only use Main.

If that was the case, the Android developers could have not introduce `LOG_ID_DEFAULT` at all. But it's there for some reason. Complying with their assumptions rather than introducing our own logic seems reasonable and future-proof.

**Comment by tyranron** [comment]
@peterdk fixed in #67, released in [0.13.1](https://github.com/Nercury/android_logger-rs/releases/tag/v0.13.1).
